react/src/pages/Home.tsx

```diff
@@ -13,18 +13,19 @@ export default () => {
   return (
     <>
       <div className="block">
-        You have entered Avenluutn, on the edge of nowhere.
+        Congratulations, newly appointed Emissary!
       </div>
       <div className="block">
-        The humble road into town smells of grass and earth.
-        Sweet wind rustles the treetops. You feel hopeful.
+        You have entered Avenluutn, a town on the edge of nowhere,
+        created by the alliance for the purpose of studying and securing the mysterious wastes.
```

Comment:
Should "the alliance" be capitalized? Otherwise I'm not sure what alliance it's referring to, I guess some alliance in this world...

Reply:
For some reason I like it better lowercase, I'm not sure why exactly. Maybe because making it uppercase feels like we're throwing too many Terms, specifying too much that we don't intend to fill in right now...

Reply:
Okay, I tried something